debian/control

```diff
@@ -40,10 +40,13 @@ Package: snapd
 Architecture: any
 Depends: adduser,
          ca-certificates,
+         cgroup-lite,
          gnupg1 | gnupg,
          snap-confine (>= 1.0.43),
          squashfs-tools,
+         sudo,
```

Comment:
I'm using the snapd package to force installation/update of certain packages. In the case of sudo, this (more precisely, a versioned dependency on sudo) forces installation of an update that adds /snap/bin to the secure path.